tsconfig-for-webpack.json

```diff
@@ -1,7 +1,16 @@
 {
+  "extends": "./tsconfig.base.json",
```

Comment:
Should this extend `tsconfig.json`? I'm not sure I understand what the difference is between the "base" version and that one.

Reply:
I think `base` mean to be extended.

maybe ask @MaikoTan here